offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/SelectItemsView.groovy

```diff
@@ -93,29 +85,29 @@ class SelectItemsView extends VerticalLayout implements Resettable{
         private String regex
 
         ProductTypeRegex(String regex) {
-            this.regex = regex;
```

Comment:
nice, happy you removed it. bothered me for some time now :D

Reply:
IntelliJ IDEA did it 😄 🤖